src/ioc-react/mapDependenciesToProps.ts

```diff
@@ -0,0 +1,17 @@
+import { Container } from '../ioc';
```

Comment:
This utility function does the heavy lifting of transforming an object of propName to diKeys into an object of propName to dependency instances. It already existed and was used by withDependencies, I just moved to this file so it can be used by both withDependencies and useDependencies